Spanner/src/SpannerClient.php

```diff
@@ -594,6 +601,19 @@ function (array $instance) {
      *     @type SessionPoolInterface $sessionPool A pool used to manage
      *           sessions.
      *     @type string $databaseRole The user created database role which creates the session.
+     *     @type array $includeReplicas Regions/Replica to be included for the read query.
+     *           If the specified replicas are not part of instance configuration,
+     *           Spanner will fallback to the default routing algorithm by selecting
+     *           the nearest healthy replica.
+     *           $includeReplicas and $excludeReplicas are mutually exclusive.
```

Comment:
nit: you may also add what happens if include and exclude replicas as not exclusive.

Reply:
latest discussion with the team says client library need to throw exception if not exclusive as well as if the size of the replica list goes above 10.

Reply:
The team confirmed library need not to throw any exceptions.

Reply:
Also, this explanation is no more there, it is not directing towards the Directed read classes